src/main/java/frc/robot/RobotContainer.java

```diff
@@ -144,15 +179,36 @@ private void configureBindings() {
 
     fieldCentricButton.onTrue(swerveDrive.toggleFieldCentric());
 
-    resetFieldCentricButton.onTrue(new InstantCommand((
-    ) -> {
+    resetFieldCentricButton.onTrue(new InstantCommand(() -> {
```

Comment:
For commands like this that doesn't actually set motor values, you should not require them, since they don't need to interrupt whatever command is running

Reply:
Is this better?